src/Apps/Artwork/Components/ArtistInfo.tsx

```diff
@@ -28,77 +27,71 @@ interface ArtistInfoProps {
   artist: ArtistInfo_artist$data
 }
 
-const Container = ({ children }) => (
-  <StackableBorderBox p={2}>{children}</StackableBorderBox>
-)
-
-@track(
-  { context_module: DeprecatedSchema.ContextModule.Biography },
-  { dispatch: data => Events.postEvent(data) }
-)
-export class ArtistInfo extends Component<ArtistInfoProps> {
-  @track({
-    action_type: DeprecatedSchema.ActionType.Click,
-    flow: DeprecatedSchema.Flow.ArtworkAboutTheArtist,
-    subject: DeprecatedSchema.Subject.ReadMore,
-    type: DeprecatedSchema.Type.Button,
-  })
-  trackArtistBioReadMoreClick() {
-    // Tracking
-  }
+const ArtistInfo: React.FC<ArtistInfoProps> = ({ artist }) => {
```

Comment:
The following app changes were migrations to React.FC, to remove our use of `@decorators` in the app. Decorators subtly break every form of tooling that i've tried historically (and they didn't work well with rspack), so finally just ripped them out. About 5 components. 

We used them for tracking, so converted things into our normal canonical tracking patterns.